File: URI/C/1048.c
#include<stdio.h>
int main()
{
    double a,b,c,d,e;

    scanf("%lf",&a);
//0 - 400.00

    if (a>=0 && a<=400) {
        printf("Novo salario: %.2lf\n",(a*0.15)+a);
        printf("Reajuste ganho: %.2lf\n",(a*0.15));
        printf("Em percentual: 15 %%\n");

    }
//400.01 - 800.00

    else if (a>=400.01 && a<=800) {
        printf("Novo salario: %.2lf\n",(a*0.12)+a);
        printf("Reajuste ganho: %.2lf\n",(a*0.12));
        printf("Em percentual: 12 %%\n");

    }
//800.01 - 1200.00

    else if (a>=800.01 && a<=1200) {
        printf("Novo salario: %.2lf\n",(a*0.10)+a);
        printf("Reajuste ganho: %.2lf\n",(a*0.10));
        printf("Em percentual: 10 %%\n");

    }
//1200.01 - 2000.00

    else if (a>=1200.01 && a<=2000) {
        printf("Novo salario: %.2lf\n",(a*0.07)+a);
        printf("Reajuste ganho: %.2lf\n",(a*0.07));
        printf("Em percentual: 7 %%\n");

    }
//Above 2000.00

    else if (a>=2000.01) {
        printf("Novo salario: %.2lf\n",(a*0.04)+a);
        printf("Reajuste ganho: %.2lf\n",(a*0.04));
        printf("Em percentual: 4 %%\n");

    }


    return 0;
}

File: URI/PYHTON/1048.py
a = float (input())

if a <= 400 :
    result = float(a*0.15)
    print('Novo salario: %.2f'%(result+a))
    print('Reajuste ganho: %.2f'%(result))
    print('Em percentual: 15 %')

elif a<= 800 :
    result = float (a * 0.12)
    print('Novo salario: %.2f'%(result+a))
    print('Reajuste ganho: %.2f'%(result))
    print('Em percentual: 12 %')

elif a <= 1200 :
    result = float (a * 0.10)
    print('Novo salario: %.2f'%(result+a))
    print('Reajuste ganho: %.2f'%(result))
    print('Em percentual: 10 %')

elif a <= 2000 :
    result = float (a * 0.07)
    print('Novo salario: %.2f'%(result+a))
    print('Reajuste ganho: %.2f'%(result))
    print('Em percentual: 7 %')

elif a >= 2000 :
    result = float (a * 0.04)
    print('Novo salario: %.2f'%(result+a))
    print('Reajuste ganho: %.2f'%(result))
    print('Em percentual: 4 %')
